commcare-cloud/commcare_cloud/commcare_cloud.py

```diff
@@ -436,17 +417,17 @@ def run_apply():
         exit(exit_code)
 
 
-class RunShellCommand(RunAnsibleModule):
+class RunShellCommand(object):
     command = 'run-shell-command'
     help = 'Run an arbitrary command via the shell module.'
 
     @staticmethod
     def make_parser(parser):
-        RunAnsibleModule._make_parser(
-            parser,
-            module_args_argument_name='shell_command',
-            module_args_argument_help="The shell command you want to run"
-        )
+        parser.add_argument('inventory_group', help=(
+            "The inventory group to run the command on. Use 'all' for all hosts."
+        ))
```

Comment:
Only minor sad part of this refactor is that this statement is copy-pasted between the two. That said, the two commands do have a sort of fundamentally different set of positional arguments, so I think it's clearer to repeat the one argument they have in common and make it explicit right here what this command's positional arguments are than to over parametrize this and make you untangle indirection to figure out what this command's args are.

Having said this, I realized the obvious, that the best of both worlds would be to pull this out into an `arg_inventory_group` helper function. I'll add that to this PR.